gcp/modules/cert-manager/main.tf

```diff
@@ -30,6 +30,6 @@ resource "null_resource" "cert_manager_resources" {
 
   provisioner "local-exec" {
     when    = "destroy"
-    command = "kubectl delete --ignore-not-found -f ${path.module}/resources/"
+    command = "kubectl delete --ignore-not-found -f ${path.module}/resources/ || true"
```

Comment:
We have `--ignore-not-found` which covers the only expected failure mode I can think of, so why do we need `|| true`?

Reply:
On forceful cluster re-creation, it tries to destroy cert-manager null resource first, and without cert-manager being deployed in the cluster, it leads to K8s failing to recognize Issuer resources:
```
* null_resource.cert_manager_resources (destroy): Error running command 'kubectl delete --ignore-not-found -f /root/.terragrunt/4BDYFL2KyB7Ei1-ELe3uiLPpHq8/exDWDCWoRRnBaZ-HQfLUjTVcEno/cert-manager/resources/ # || true': exit status 1. Output: unable to recognize "/root/.terragrunt/4BDYFL2KyB7Ei1-ELe3uiLPpHq8/exDWDCWoRRnBaZ-HQfLUjTVcEno/cert-manager/resources/issuers.yaml": no matches for kind "ClusterIssuer" in version "certmanager.k8s.io/v1alpha1"
```